code/__DEFINES/mobs.dm

```diff
@@ -63,7 +58,49 @@
 #define ALIEN_BODYPART "alien"
 #define LARVA_BODYPART "larva"
 
-
+//Bodypart change blocking flags
+#define BP_BLOCK_CHANGE_SPECIES (1<<0)
+
+//Bodytype defines for how things can be worn, surgery, and other misc things.
+#define BODYTYPE_ORGANIC (1<<0)
+#define BODYTYPE_ROBOTIC (1<<1)
+#define BODYTYPE_HUMANOID (1<<2) //Everything
+#define BODYTYPE_DIGITIGRADE (1<<3) //Cancer
+
+//Defines for Species IDs
+#define SPECIES_ABDUCTOR "abductor"
+#define SPECIES_ANDROID "android"
+#define SPECIES_CORPORATE "corporate"
```

Comment:
corporates were removed

Reply:
probably should just copy over the old list from DNA.dm

Reply:
the ids are deliberately chosen to match those of their limbs, so I can't do that. This pr is half finished I wasn't expecting so many reviews with it marked as a draft